.github/workflows/package.yml

```diff
@@ -0,0 +1,30 @@
+name: Package
+on:
+  push: {}
+jobs:
+  package:
+    name: Package
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+
+      - name: Install libraries
+        run: sudo apt-get update && sudo apt-get install -y libpam0g-dev
+
+      - name: Install nfpm
+        run: go install github.com/goreleaser/nfpm/v2/cmd/nfpm@v2.6.0
+
+      - name: Build RPM
+        run: make rpm
+
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        if: startsWith(github.ref, 'refs/tags/')
```

Comment:
would it make sense to check that `hack/package_version.sh != "dev"`

Reply:
derp, never mind

Reply:
We're pretty limited to what we can do in an `if` here. But in general, a release is 1:1 with a tag, so I think it's best to check for a tag here.